Sources/SwiftFCSDKiOS/SwiftFCSDKiOS.docc/PictureinPicture.md

```diff
@@ -1,5 +1,98 @@
 # Picture in Picture
 
+FCSDKiOS offers Picture in Picture support. In order to use this feature you will need to be able to conform to **AVPictureInPictureControllerDelegate** and/or **AVPictureInPictureSampleBufferPlaybackDelegate**. The **SDK** consumer is completely responsible for conforming to these delegates. However you will need to get a layer or view from the **SDK** to present in the **AVPictureInPicture** code.
+
+## Overview
+
+The Picture in Picture feature is set up in the consumer of the **SDK**, however **FCSDKiOS** gives you all of the needed pieces in order to build a PiP app. 2 things are required to use Picture in Picture mode:
+
+1. Your application must be using **FCSDKiOS**'s Preview Buffer View
+2. Your application must be using **FCSDKiOS**'s Sample Buffer View
+
+If you have conformed to these 2 **UIView**s please see how to set up Picture in Picture below.
+
+### AVCaptureSession
+In order to check if your devices support Multitasking Camera Access, when a **PreviewBufferView** is created we can get the **AVCaptureSession** from the **PreviewBufferView**. If your Devices is supported by Apple, you can use the new **AVPictureInPictureVideoCallViewController** provided by Apple. Otherwise, you will need to continue using **AVPictureInPictureController** and request permission from Apple for the proper entitlement. This method only works if you are using **PreviewBufferView** for your Local Video UIView.
```

Comment:
There is at least a grammar error here: "If your Devices is supported by Apple" probably should read "If your devices are supported by Apple" if that is the meaning intended here.